src/app.tsx

```diff
@@ -19,25 +19,25 @@
 
 import React, { useContext, useEffect, useMemo, useState } from "react";
 
-import { Page, PageSection } from "@patternfly/react-core/dist/esm/components/Page";
-import { Sidebar, SidebarPanel, SidebarContent } from "@patternfly/react-core/dist/esm/components/Sidebar";
 import {
     AlertGroup, Alert, AlertVariant, AlertActionCloseButton
 } from "@patternfly/react-core/dist/esm/components/Alert";
+import { Page, PageSection } from "@patternfly/react-core/dist/esm/components/Page";
+import { Sidebar, SidebarPanel, SidebarContent } from "@patternfly/react-core/dist/esm/components/Sidebar";
 import { ExclamationCircleIcon } from "@patternfly/react-icons";
 
 import cockpit from "cockpit";
```

Comment:
From my understanding, we prefer this to be separate. (Although it is imported from `pkg/lib` obviously.

Reply:
Not a real blocker from my side, it does look nicer though.

Reply:
I recently started to lump it together with other pkg/lib modules.